src/latexify/codegen/function_codegen.py

```diff
@@ -162,21 +162,47 @@ def visit_Match(self, node: ast.Match) -> str:
             if i < len(node.cases) - 1:
                 body_latex = self.visit(case.body[0])
                 cond_latex = self.visit(case.pattern)
-                case_latexes.append(
-                    body_latex + r", & \mathrm{if} \ " + subject_latex + cond_latex
-                )
+                if case.guard is not None:
+                    cond_latex = self._expression_codegen.visit(case.guard)
```

Comment:
This doesn't generate correct expression: e.g., if we get `x, y if x == y`, it generates $\mathrm{if} \ x = y$, but $x$ and $y$ are not defined. It is better to avoid supports for guards from this PR and make additional discussions.